### PR TITLE
Write a History section to replace Prior Art, JSON and XML

### PR DESCRIPTION
**Pull request by ingydotnet**
The replaced sections are no longer as relevant as they were 20 years
ago.

The new History section touches upon the key points, and hopefully is a
useful and interesting intro to people new to YAML.

### CODE REVIEW

File: spec/.aspell [hunk 1]
```diff
@@ -57,6 +60,11 @@ interoperable
 interprocess
 json
 kramdown
+<<<<<<< HEAD
```
Comment (Thom1729) [inline spec/.aspell:63]: Conflict?

File: spec/spec.md [hunk 2]
```diff
@@ -131,126 +131,54 @@ The design goals for YAML are, in decreasing priority:
...
-[https://yaml.org/xml](/xml).
+## #. History
+
+YAML design began in early 2001, when Clark Evans and Oren Ben-Kiki were
+working with the SML-DEV[^SML-DEV] mailing list which was focused on
+simplifying XML.
+They were introduced to Ingy döt Net who was working on a plain text
+serialization module[^Denter] for Perl.
+The three of them collaborated through the yaml-core mailing list[^yaml-core]
+to create the YAML 1.0 specification, which was published in early 2004.
```
Comment (Thom1729) [inline spec/spec.md:142]: Should probably link the YAML 1.0 spec.

File: spec/spec.md [hunk 3]
```diff
@@ -131,126 +131,54 @@ The design goals for YAML are, in decreasing priority:
...
+
+YAML took many of its ideas from existing technologies such as C, Java, Perl,
+Python, Ruby, Email, HTML, MIME, URIs, XML, SAX, SOAP and JSON.
+
+Adoption of YAML happened in a grassroots manor during this time.
```
Comment (Thom1729) [inline spec/spec.md:147]: manor → manner

File: spec/spec.md [hunk 4]
```diff
@@ -131,126 +131,54 @@ The design goals for YAML are, in decreasing priority:
...
+YAML design began in early 2001, when Clark Evans and Oren Ben-Kiki were
+working with the SML-DEV[^SML-DEV] mailing list which was focused on
+simplifying XML.
+They were introduced to Ingy döt Net who was working on a plain text
+serialization module[^Denter] for Perl.
+The three of them collaborated through the yaml-core mailing list[^yaml-core]
+to create the YAML 1.0 specification, which was published in early 2004.
+
+YAML took many of its ideas from existing technologies such as C, Java, Perl,
+Python, Ruby, Email, HTML, MIME, URIs, XML, SAX, SOAP and JSON.
```
Comment (Thom1729) [inline spec/spec.md:145]: Probably lowercase “Email”. Without the link to the specific standard, it looks weird in uppercase.

File: spec/spec.md [hunk 5]
```diff
@@ -131,126 +131,54 @@ The design goals for YAML are, in decreasing priority:
...
+Ruby was the first language to ship a YAML framework as part of its core.
+
+The YAML 1.1 specification was published in 2005.
+Around this time, the developers became aware of JSON[^JSON].
+By sheer coincidence, JSON was almost a complete subset of YAML (both
+syntactically and semantically).
+This was due to YAML's flow collection style, which was influenced by Python
+more than JavaScript.
+
+In 2006, Kirill Siminov produced PyYAML and libyaml, both of which are still
```
Comment (Thom1729) [inline spec/spec.md:157]: Should link.

Comment (ingydotnet) [reply spec/spec.md:157]: Links added.

File: spec/spec.md (hunk 4 above)
Comment (ingydotnet) [reply spec/spec.md:145]: done

File: spec/spec.md (hunk 3 above)
Comment (ingydotnet) [reply spec/spec.md:147]: fixed

File: spec/.aspell (hunk 1 above)
Comment (ingydotnet) [reply spec/.aspell]: fixed

**Review by Thom1729 — APPROVED** [review]

**Review by perlpunk — APPROVED** [review]

**Pull request merged**